website/apps/glossary/views.py

```diff
@@ -1,9 +1,13 @@
+from django.http import HttpResponseRedirect
 from django.shortcuts import render_to_response
 from django.template import RequestContext
 from backend.api import Glossary
 
 
 def glossary(request):
+    if request.mission.memorial:
+        return HttpResponseRedirect('/')
```

Comment:
This repetition bothers me a bit. Could we use a `TemplateView` here, so we can re-use the `MemorialMixin`?

I'm generally in favour of using CBVs everywhere anyway, because they're more easily extensible, and they have nicer defaults (e.g. 405s).